catalog/app/utils/Delay.tsx

```diff
@@ -1,6 +1,12 @@
 import * as React from 'react'
 
-export default function Delay({ ms = 1000, alwaysRender = false, children }) {
+interface DelayProps {
+  ms?: number
+  alwaysRender?: boolean
+  children: (ready?: boolean) => JSX.Element
```

Comment:
I think, using `React.ReactElement` is better, because `React` is imported here

Reply:
i would use ReactNode, but it doesnt compile, react typings for functional components are a little funky (probably for historical reasons)